src/index.html

```diff
@@ -1,13 +1,53 @@
-<!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+<!Doctype html>
+<html lang="en" class="page">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+    <title>Product cards</title>
+    <link rel="stylesheet" href="styles/index.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="container">
+      <div class="card container__card" data-qa="card">
+        <img
+        class="card__image"
+        src="images/imac.jpeg"
+        alt="APPLE A1419 iMac 27"
+        >
+        <h2 class="card__title">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h2>
+
+        <div class="card__code">Код товара: 195434</div>
+
+        <div class="card__rating">
+          <div class="stars stars--4">
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+          </div>
+
+          <div class="card__review">
+            Отзывов: 5
+          </div>
+
+        </div>
+
+        <div class="card__price">
+          <div class="card__name">Цена:</div>
+          <div class="card__number">69 999 грн</div>
+        </div>
+
+        <a
+        href="#"
+        class="card__buy"
+        data-qa="hover"
+        >Купить</a>
```

Comment:
Here too, set the correct indents

```suggestion
        <a
          href="#"
          class="card__buy"
          data-qa="hover"
        >
        Купить
        </a>
```